src/hooks/useGhostBlogPosts.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from "react"
+// @ts-ignore
+import GhostContentAPI from "@tryghost/content-api"
+
+const useGhostBlogPosts = () => {
+  const [blogPosts, setBlogPosts] = useState([])
+
+  const api = new GhostContentAPI({
+    url: "https://threshold-network.ghost.io",
+    // API key is not private and is safe to use on frontend:
+    // https://ghost.org/docs/content-api
+    key: "ca1ac53ba62d6f30b55b81890e",
+    version: "v3",
+  })
+
+  useEffect(() => {
+    api.posts
+      .browse({ limit: 5, include: "tags,authors" })
+      .then((posts: any) => {
+        setBlogPosts(
+          posts.map((post: any) => {
+            const date = new Date(post.published_at).toLocaleDateString(
```

Comment:
@r-czajkowski I am planning to use the new date util I created for the other page. Can update once these PR's are merged.